onboarding_project/squad_pantry_app/tests.py

```diff
@@ -5,8 +5,10 @@
 
 class OrderTestCase(TestCase):
     def setUp(self):
-        self.user1 = SquadUser.objects.create(username="prakhar", password="testdb1", is_superuser=True, is_kitchen_staff=False)
-        self.user2 = SquadUser.objects.create(username="prakhar2", password="testdb2", is_superuser=True, is_kitchen_staff=False)
+        self.user1 = SquadUser.objects.create(username="prakhar", password="testdb1",
+                                              is_superuser=True, is_kitchen_staff=False)
+        self.user2 = SquadUser.objects.create(username="prakhar2", password="testdb2",
+                                              is_superuser=True, is_kitchen_staff=False)
 
     def test_cancel_order(self):
```

Comment:
No unit test for `check_limit`? All core business logic should have unit tests

Reply:
Done!